ecl/hqlcpp/hqlcerrors.hpp

```diff
@@ -224,6 +224,7 @@
 #define HQLERR_BodyNotAllowedWithInline         4204
 #define HQLERR_DatasetPassedToRowArg            4205
 #define HQLERR_HashStoredDuplication            4207
+#define HQLERR_RankOnNonList                    4208
```

Comment:
@ghalliday No associated HQLERR_RankOnNonList_Text.  Also, this is unused for now.